cypress/tests/data/60-content/VkarbasizaedSubmission.cy.js

```diff
@@ -135,11 +135,10 @@ describe('Data suite tests', function() {
 		cy.get('[data-cy="sidemodal-header"]').contains("Published");
 		// check publication status
 		cy.get('[data-cy="workflow-controls-left"]').contains("Published");
-		cy.contains('This version has been published and can not be edited.');
```

Comment:
I'm not sure if the new warning text appears here -- but if it does, there should be a Cypress check for it.